lib/constants.ts

```diff
@@ -16,6 +16,9 @@ export const TEST_RUNNER_NAME = "nativescript-unit-test-runner";
 export const LIVESYNC_EXCLUDED_FILE_PATTERNS = ["**/*.js.map", "**/*.ts"];
 export const XML_FILE_EXTENSION = ".xml";
 export const PLATFORMS_DIR_NAME = "platforms";
+export const IOS_PLATFORM_NAME = "ios";
+export const IOS_PLATFORM_NORMALIZED_NAME = "iOS";
```

Comment:
I'll do it, although there are already some constants in ios-project-service for "ios" and it turned out my code pretty ugly from:
`constants.IOS_PLATFORM_NORMALIZED_NAME`
to:
`this.$mobileHelper.normalizePlatformName(this.$devicePlatformsConstants.iOS)`